cmd/res/device.virtual.binary.yaml

```diff
@@ -10,28 +10,13 @@ deviceResources:
     name: "Binary"
     description: "Generate random binary value"
```

Comment:
specify `isHidden: false` for all the normal Device Resource and Device Command because this is a sample profile